src/test/java/seedu/address/ui/testutil/GuiTestAssert.java

```diff
@@ -51,6 +58,37 @@ public static void assertCardDisplaysMedicine(Medicine expectedMedicine, Medicin
         assertEquals(expectedMedicine.getStock().value.toString(), actualCard.getStock());
     }
 
+    /**
+     * Returns the color style for {@code tagName}'s label. The tag's color is determined by looking up the color
+     * in {@code PersonCard#TAG_COLOR_STYLES}, using an index generated by the hash code of the tag's content.
+     *
+     * @see PersonCard#getTagColorStyleFor(String)
+     */
+    private static String getTagColorStyleFor(String tagName) {
```

Comment:
Using a `String` in a `switch` statement may not be the best practice as a small typo error can potentially break the logic. You might want to use variables instead to represent your `String`.

```java
private static String BLUE_CHAS_COLOUR = "blue";
private static String ORANGE_CHAS_COLOUR = "orange";

etc.
```